2.5/test/run

```diff
@@ -138,6 +138,18 @@ test_scl_usage() {
   fi
 }
 
+test_from_dockerfile() {
+  info "Check building using a Dockerfile"
+  mydir=$(readlink -zf ${image_dir}/..)
+  ct_test_app_dockerfile $mydir/examples/from-dockerfile/2.5/Dockerfile 'https://github.com/sclorg/rails-ex.git' 'Welcome to your Rails application on OpenShift' app-src
```

Comment:
I'm afraid that's exactly not what I meant, as you're duplicating the code into two methods when one can be used instead:

```suggestion
  append="${1:-}"
  root_dir=$(readlink -zf ${image_dir}/..)
  version="$(basename "${image_dir}")"
  info "Check building using a Dockerfile"
    ct_test_app_dockerfile "${root_dir}/examples/from-dockerfile/${version}/Dockerfile${append}" 'https://github.com/sclorg/rails-ex.git' 'Welcome to your Rails application on OpenShift' app-src
```